aws/resource_aws_redshift_snapshot_copy_grant_test.go

```diff
@@ -0,0 +1,66 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAWSRedshiftSnapshotCopyGrant_Basic(t *testing.T) {
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSRedshiftSnapshotCopyGrantDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSRedshiftSnapshotCopyGrant_Basic("basic"),
```

Comment:
The random name being passed in should be randomized to prevent issues with concurrent testing or a failed run, e.g. 

```go
rName := acctest.RandomWithPrefix("tf-acc-test")
// ...
testAccAWSRedshiftSnapshotCopyGrant_Basic(rName)
// ...
resource.TestCheckResourceAttr("aws_redshift_snapshot_copy_grant.basic", "snapshot_copy_grant_name", rName),
```